website/About.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+<meta charset="UTF-8">
+<title>Joe Johnson|Portfolio</title>
+<link rel="stylesheet" href="css/main.css">
+    </head>
+
+  <body>
+
+<hr / class="bold-line-border"> 
+
+
+    <header>

```

Comment:
indent your code to make it easier to read

here is a link to more information about indentation
https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html